src/plugins/imagery/components/ImageryViewLayout.vue

```diff
@@ -0,0 +1,274 @@
+                this.unselectAllImages();
+                this.isPaused = state;
+
+                if (this.nextDatum) {
+                    this.updateValues(this.nextDatum);
+                    delete this.nextDatum;
+                } else {
+                    this.updateValues(this.imageHistory[this.imageHistory.length - 1]);
+                }
+
+                this.autoScroll = true;
+            }
+
+            return this.isPaused;
+        },
+        requestHistory(bounds) {
+            this.requestCount++;
+            this.imageHistory = [];
+            const requestId = this.requestCount;
+            this.openmct.telemetry
+                .request(this.domainObject, bounds)
+                .then(function (values = []) {
+                    if (this.requestCount > requestId) {
+                        return Promise.resolve('Stale request');
+                    }
+
+                    values.forEach(function (datum) {
+                        this.updateHistory(datum);
+                    }, this);
+
+                    this.updateValues(values[values.length - 1]);
+                }.bind(this));
+        },
+        scrollToBottom() {
+            if (this.isPaused || !this.$refs.thumbsWrapper || !this.autoScroll) {
+                return;
+            }
+
+            const scrollHeight = this.$refs.thumbsWrapper.scrollHeight || 0;
+            if (!scrollHeight) {
+                return;
+            }
+
+            setTimeout(() => this.$refs.thumbsWrapper.scrollTop = scrollHeight, 0);
+        },
+        setSelectedImage(image) {
+            this.imageUrl = this.getImageUrl(image);
+            this.time = this.getTime(image);
+            this.paused(true);
+            this.unselectAllImages();
+            image.selected = true;
+        },
+        getElSize() {
+            if (!this.$el) {
+                return;
+            }
+
+            this.elementHeight = this.$el.parentElement.offsetHeight;
+            return this.elementHeight;
+        },
+        stopListening() {
+            if (this.unsubscribe) {
+                this.unsubscribe();
+                delete this.unsubscribe;
+            }
+        },
+        subscribe(domainObject) {
+            this.date = ''
+            this.imageUrl = '';
+            this.openmct.objects.get(this.keystring)
+                .then(function (object) {
+                    const metadata = this.openmct
+                        .telemetry
+                        .getMetadata(this.domainObject);
+                    this.timeKey = this.openmct.time.timeSystem().key;
+                    this.timeFormat = this.openmct
+                        .telemetry
+                        .getValueFormatter(metadata.value(this.timeKey));
+                    this.imageFormat = this.openmct
+                        .telemetry
+                        .getValueFormatter(metadata.valuesForHints(['image'])[0]);
```

Comment:
Having `this.openmct.telemetry.getValueFormatter` and `this.openmct.telemetry.getMetadata` in one line improves readability and reduces lines of code.

Reply:
yeah I think this was due to 'lint:fix' :(